### DISCUSSION TITLE
Searching for a single word doesn't find anything

### DISCUSSION THREAD

**Discussion by rillig** [community/community#10990] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.com/llvm/llvm-project/search?q=warn_concatenated_literal_array_init

https://github.com/llvm/llvm-project/blob/4111fafa5b8810887a3465e67937681df1a7342a/clang/lib/Sema/SemaDecl.cpp#L13305 contains that word. Why is this occurrence not found?

**Comment by colinwm** [comment, marked as answer]
Hey @rillig, unfortunately right now we're only indexing files that are smaller than 350 kB. That file is >700 kB! To see a full list of limitations on what we index, you can read our FAQ: https://cs.github.com/about/faq#indexed-content

Sorry about that! We're working on indexing improvements to hopefully expand what we can index, but for now that's the limitation.

**Reply by rillig** [reply]
Thank you for your answer, that explains it all.

If it should turn out that "why doesn't the search find my code" becomes an FAQ, would it make sense to add a note to the empty search result? In my case I searched a single repository and didn't find the word at all, so I know this is a special case. I don't know whether this case happens often enough to warrant a note. Maybe there could be a note on projects that contain big files?
